tokio/src/net/tcp/listener.rs

```diff
@@ -12,8 +12,22 @@ use std::task::{Context, Poll};
 cfg_tcp! {
     /// A TCP socket server, listening for connections.
     ///
+    /// Also implements a stream over the connections being received on this listener.
+    ///
+    /// The stream will never return `None` and will also not yield the peer's
+    /// `SocketAddr` structure. Iterating over it is equivalent to calling accept in a loop.
```

Comment:
I think the sentence structure here is a bit awkward; in particular starting a sentence with "Also" irks me.